modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -383,7 +383,8 @@ package object engine {
     modify(st => Engine.State(st.conditions, st.operator, st.sequences.updated(id, s)))
 
   // For debugging
-  def printSequenceState(id: Sequence.Id): HandleP[Option[Unit]] = getSs(id)((qs: Sequence.State) => Task.now(println(qs)).liftM[HandleStateT])
+  def printSequenceState(id: Sequence.Id): HandleP[Unit] =
+    getSs(id)((qs: Sequence.State) => Task.now(println(qs)).liftM[HandleStateT]).void
```

Comment:
No, I just split them whet they seem too long. I'm not sure if gem's scalatyle settings flag them